config-provisioning/src/main/java/com/yahoo/config/provision/CloudName.java

```diff
@@ -55,7 +53,7 @@ public String toString() {
     }
 
     @Override
-    public int compareTo(@NotNull CloudName o) {
+    public int compareTo(CloudName o) {
```

Comment:
Should we do this case insensitive, or document the preferred casing?

Reply:
Not sure, for now it follows what we've done in `RegionName`: https://github.com/vespa-engine/vespa/blob/d45993e8308b0b9cb586024923878af51c98a688/config-provisioning/src/main/java/com/yahoo/config/provision/RegionName.java#L54 which is also case-sensitive. The preferred casing would need to be documented internally then, since this would be our preference...